### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by gregdolby**
This is a a fully converted TS SDK which builds/tests/starts and lints.

@Santiago-Souto could you take a good look over TransformWorker.worker.ts and Codecs.ts. I tried my best at typing those files using WebRTC type from MDN and reverse engineering the data in these files but I'm still not 100%.

### CODE REVIEW

File: packages/millicast-sdk/src/View.ts [hunk 1]
```diff
@@ -467,13 +479,14 @@ export default class View extends BaseWebRTC {
...
 
-  async onRtcDrmFetch(url: string, opts: any) {
-    if (!opts.headers) {
-      opts.headers = new Headers()
-    }
+  async onRtcDrmFetch(url: string, opts: RequestInit) {
+    opts.headers = (opts.headers as Headers) || new Headers()
+    // if (!opts.headers) {
+    //   opts.headers = new Headers()
+    // }
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/View.ts:489]: Is this meant to be left commented out?

File: packages/millicast-sdk/src/utils/Codecs.ts [hunk 2]
```diff
@@ -64,28 +38,43 @@ export const DOLBY_SEI_TIMESTAMP_UUID = '9a21f3be-31f0-4b78-b0be-c7f7dbb97250'
 // When the SDK inserts its own timecode into the unregistered block, it uses this identifier
 export const DOLBY_SDK_TIMESTAMP_UUID = 'd40e38ea-d419-4c62-94ed-20ac37b4e4fa'
 
+export interface SeiMetadata {
+  seiPicTimingTimeCodeArray: SeiPicTimingTimeCode[]
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/utils/Codecs.ts:42]: Shouldn't this be optional? As it is being set in a switch case

File: packages/millicast-sdk/src/View.ts [hunk 3]
```diff
@@ -466,13 +479,14 @@ export default class View extends BaseWebRTC {
...
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/View.ts:489]: Is this intended to be commented out?

**Review by Santiago-Souto — CHANGES_REQUESTED** [review]

File: packages/millicast-sdk/src/View.ts (hunk 3 above)
Comment (gregdolby) [reply packages/millicast-sdk/src/View.ts:489]: Nope, good catch

File: packages/millicast-sdk/src/utils/Codecs.ts (hunk 2 above)
Comment (gregdolby) [reply packages/millicast-sdk/src/utils/Codecs.ts:42]: I canged this to optional.

**Review by Santiago-Souto — APPROVED** [review]
Sample apps are not working but probably due to being in TS not fully converted

**Comment by gregdolby** [comment]
> Sample apps are not working but probably due to being in TS not fully converted

Sample apps should be working. Try runing 'nx reset' delete node_modules and package-lock. Also be sure to delete all node_modules in pacakges. The only node_modules that exist any more are in project root. I just validated that the apps work

**Pull request merged**